### DISCUSSION TITLE
关于请求失败

### DISCUSSION THREAD

**Discussion by lingheaila** [mikumifa/BiliShareMall#2] [Q&A] [answered]
操作系统：win11 23H2 22631.4541 
包：BiliShareMall-windows-amd64 v0.1.7
安装完成扫码登录后提示请求失败
路径全英文无空格，卸载重装默认路径依旧如此

![image](https://github.com/user-attachments/assets/c6273fdd-0ea6-45c8-80ce-b49e93dfa5eb)
求解

**Comment by mikumifa** [comment, marked as answer]
刚刚更新一下，应该解决了

**Reply by lingheaila** [reply]
感谢！
